ledger/simulation/trace.go

```diff
@@ -135,6 +141,13 @@ func validateSimulateRequest(request Request, developerAPI bool) error {
 			},
 		}
 	}
+	if !request.TraceConfig.Enable && request.TraceConfig.Stack {
+		return InvalidRequestError{
+			SimulatorError{
+				err: fmt.Errorf("the request didn't ask for enabling returning basic trace, but ask for returning stack change"),
+			},
+		}
+	}
```

Comment:
Perhaps this is an answer to my question about `goal` - you'll get the error reported this way.